docs/api/users-create-user.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+---
+
+# Create user
+
+Creates a [`user`](user) who is registering with the to-do service. The request body contains the new user details. You must specify the required properties for the user. 
+
+## URL
+
+```shell
+
+{POST}{server_url}/users/
+```
+
+## Request headers
+
+This request does not use any authorization. The endpoint is available to all users and applications.
+
+| Header name | Description | Required | Values |
+| -------------- | ------ | ------------ |------------ |
+| Content-Type | The format of the data to be posted. | Optional | application/json. Default value.  |
+| Accept | The format of the data to be returned. | Optional | application/json. Default value.|
+
+## Request body
+
+In the request body, specify a JSON representation of the [`user`](user) object. The following table lists the properties that are required when you create a user. 
+
+| Property | Description | Type | Required | Notes |
+| -------------- | ------ | ------------ |------------ |------------ |
+| last_name | The user's last name. | string | Required |   |
+| first_name | The user's first name. | string | Required |  |
+| email | The user's email. | string | Required | A unique email is required. |
+
+
+## Sample request
+
+The POST body should look something like this. You can change the values of each property as you’d like.
+
+```js
+[
+    {
+        "last_name": "Coughlan",
+        "first_name": "Sinead",
+        "email": "scough2@uw.edu",
+    }
+]
+```
+
+## Return body
+The following example shows the response. Note that the names should be the same as you used in your **Request body** and the response should include the new user’s id. The user’s id is automatically generated when the user is created.
+
+```js
+[
+    {
+        "id": "f0d3",
+        "last_name": "Coughlan",
+        "first_name": "Sinead",
+        "email": "scough2@uw.edu"
+    }
+]
+```
+
+## Return status
+
+| Status value | Return status | Description |
+| ------------- | ----------- | ----------- |
+| 201 | Created | User data created successfully. |
+| 500 | Internal server Error | Invalid JSON. |
+| ECONNREFUSED | N/A | Service is offline. Start the service and try again. |
+
+For more information about error handling, see  [Error handling codes](link to Mary Frances topic, tbd). 
```

Comment:
```suggestion
```
See next comment.

Reply:
Thanks for your feedback, Bob, which makes perfect sense! I've committed the changes you suggested to users-create-user.md. I trust those two updates are in order from your perspective? 

Regarding the to-do-db_source.json file, yes, that was an oversight on my part. I'd been testing it previously and somehow it got included in the PR.  Can you advise me how to remove it from the PR? Thank you.

Reply:
I see that I've removed the culprit json file in my latest squash > force push commit. Thanks for instructions. All good to merge, now?